deploy/src/main/deploy/example-permissions.json

```diff
@@ -70,6 +74,10 @@
       {
         "operation": "registration/*:*",
         "activities": [ "EXECUTE" ]
+      },
+      {
+        "resource": "notification/*",
+        "activities": [ "READ" ]
```

Comment:
Good point!
I've had a look at the corresponding `permissions.json` file in the `tests` module (also not defining the WRITE activity).
The device registry (and also the protocol adapters) connect to Qpid on port 5673. The corresponding listener in the qpid configuration isn't associated with the auth plugin, it is configured with `"saslMechanisms": "EXTERNAL"` instead.

So the above `notification/*` resource in the `permissions.json` isn't actually used (because Qpid doesn't check with the Hono auth server here). The same applies to the `telemetry/*`, `event/*` resources for the protocol-adapter role (WRITE for `command_response/*` is also missing).

Isn't it, that we would need to switch to Basic Auth for the device-registry/adapter Qpid connections to enable this kind of check here?

Reply:
Yes, the Auth Plugin cannot be used with SASL EXTERNAL. I suggest to remove the unused/unneeded authorizations from this file then.

Reply:
I've removed the added authorizations. I'll remove the others outside of this PR.

Reply:
For the integration test, there is a resource entry needed in `permissions.json` (I previously used the adapter user here).
I've added a separate user/role now.